nautobot_ssot_infoblox/diffsync/models/nautobot.py

```diff
@@ -87,6 +90,39 @@ def create(cls, diffsync, ids, attrs):
             status=status,
             description=attrs.get("description", ""),
         )
+        if attrs.get("vlans") and len(attrs["vlans"].keys()) >= 1:
+            relationship_dict = {
```

Comment:
isn't this relationship good enough?
https://github.com/nautobot/nautobot/blob/develop/nautobot/ipam/models.py#L469

Reply:
The Relationship is for when there are more than one VLAN assigned to a Subnet. The only way to document that is with a Relarionship.